lib/tbot/service_ssh_multiplexer.go

```diff
@@ -539,12 +540,47 @@ func (s *SSHMultiplexerService) handleConn(
 		}
 	}()
 
+	var stderr *os.File
+	defer func() {
+		if stderr == nil {
+			return
+		}
+		defer stderr.Close()
+		if err != nil {
+			fmt.Fprintln(stderr, err)
+		}
+	}()
+
+	var req string
+	// here we try receiving a file descriptor to use for error output, which
+	// should be mapped to OpenSSH's own stderr (or /dev/null)
+	if un, _ := downstream.(*net.UnixConn); un != nil {
+		b := make([]byte, 1)
+		fds := make([]*os.File, 1)
+
+		// TODO(espadolini): get rid of [uds.Conn]
```

Comment:
Does this TODO need addressing ?

Reply:
Not in this PR, but there's only drawbacks to having a wrapper around `*net.UnixConn`, especially if what we're doing with them is `io.Copy`ing data around.